lib/discord-client/events/commands/commands/ramble.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const _ = require('lodash');
+const axios = require('axios');
+const config = require('config');
+const fs = require('fs');
+
+const loadRamblings = _.once(async () => {
+  const source = _.get(config, 'discord.commands.ramble.ramblings');
+  if (_.isArray(source)) return source;
+  if (_.startsWith(source, 'https://')) {
```

Comment:
Non TLS is for the weak!

Reply:
`isSecure = _.includes(addressBar, "🔒");`